.github/workflows/coverage.yml

```diff
@@ -0,0 +1,66 @@
+name: API Coverage
+
+on: [push, pull_request]
+
+# git clone git@github.com:dblock/OpenSearch.git
+# cd OpenSearch
+# git remote add upstream git@github.com:opensearch-project/OpenSearch.git
+# git fetch upstream --tags
+# git checkout 2.11.1
+# git checkout -b 2.11.1-api-plugin
+# git cherry-pick ffa9ce9d41ac4d8a15b7041b4c8c60c4d3b1cffc
+# git push origin 2.11.1-api-plugin
+
+env:
+  JAVA_VERSION: 11
+  OPENSEARCH_VERSION: 2.11.1
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.JAVA_VERSION }}
+      - name: Checkout OpenSearch
+        uses: actions/checkout@v2
+        with:
+          repository: 'dblock/OpenSearch'
```

Comment:
@dblock we should probably merge that after https://github.com/opensearch-project/OpenSearch/pull/11843/files, right?